testnet/start-obscuro-node.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+#
+# This script downloads and builds the obscuro node
```

Comment:
@otherview Is this comment outdated? No downloading here AFAICT

Reply:
It downloads the docker container images.